packages/mobile/src/components/core/IconButton.tsx

```diff
@@ -37,6 +40,7 @@ const defaultHitSlop = { top: 12, right: 12, bottom: 12, left: 12 }
  * The default size is 18x18 but this can be overridden via styles.icon
  */
 export const IconButton = ({
+  autoWidth = false,
```

Comment:
cool updates here! just want to make sure this leads to consistent results with the other icon buttons?

Reply:
Yea this should only affect buttons that set this to true so it should be ok

Reply:
ah right of course!

Reply:
oh one thing actually, so we have `fullWidth` on buttons that might do the same thing, can we use that prop name for consistency?